kdtree/include/pcl/kdtree/kdtree_flann.h

```diff
@@ -178,7 +178,7 @@ namespace pcl
         * \exception asserts in debug mode if the index is not between 0 and the maximum number of points
         */
       int 
-      radiusSearch (const PointT &point, double radius, std::vector<int> &k_indices,
+      radiusSearch (const PointT &point, double radius, Indices &k_indices,
                     std::vector<float> &k_sqr_distances, unsigned int max_nn = 0) const override;
```

Comment:
should max_nn be index_t as well - since its the max numbers of indexes?

Reply:
I'm touching other modules enough to just compile in 64 bit mode on Linux